CHANGELOG.md

```diff
@@ -1,9 +1,104 @@
+| `int_sap__purchasing_document_status` | New model |  |  | Intermediate model for purchasing document statuses (`dd07l`). |
+| `int_sap__purchasing_document_type` | New model |  |  | Intermediate model for purchasing doc types (`t161`). |
+| `int_sap__purchasing_group` | New model |  |  | Intermediate model for purchasing groups (`t024`). |
+| `int_sap__purchasing_organization` | New model |  |  | Intermediate model for purchasing organizations (`t024e`). |
+| `int_sap__sales_document_header` | New model |  |  | Intermediate model for sales document headers (`vbak`). |
+| `int_sap__sales_document_header_status` | New model |  |  | Intermediate model for sales doc header statuses (`vbuk`). |
+| `int_sap__sales_document_item` | New model |  |  | Intermediate model for sales document items (`vbap`). |
+| `int_sap__sales_document_item_status` | New model |  |  | Intermediate model for sales doc item statuses (`vbup`). |
+| `int_sap__sales_document_rejection_reason` | New model |  |  | Intermediate model for sales rejection reasons (`tvag`). |
+| `stg_sap__dd07l` | New model |  |  | Staging model for domain values (`dd07l`). |
+| `stg_sap__dd07t` | New model |  |  | Staging model for domain value descriptions (`dd07t`). |
+| `stg_sap__ekbe` | New model |  |  | Staging model for purchasing document history (`ekbe`). |
+| `stg_sap__eket` | New model |  |  | Staging model for purchasing schedule lines (`eket`). |
+| `stg_sap__ekko` | New model |  |  | Staging model for purchasing document headers (`ekko`). |
+| `stg_sap__ekpo` | New model |  |  | Staging model for purchasing document items (`ekpo`). |
+| `stg_sap__makt` | New model |  |  | Staging model for material descriptions (`makt`). |
+| `stg_sap__t001w` | New model |  |  | Staging model for plants (`t001w`). |
+| `stg_sap__t024` | New model |  |  | Staging model for purchasing groups (`t024`). |
+| `stg_sap__t024e` | New model |  |  | Staging model for purchasing organizations (`t024e`). |
+| `stg_sap__t134` | New model |  |  | Staging model for material types (`t134`). |
+| `stg_sap__t134t` | New model |  |  | Staging model for material type descriptions (`t134t`). |
+| `stg_sap__t161` | New model |  |  | Staging model for purchasing document types (`t161`). |
+| `stg_sap__t161t` | New model |  |  | Staging model for purchasing document type descriptions (`t161t`). |
+| `stg_sap__tvag` | New model |  |  | Staging model for sales document rejection reasons (`tvag`). |
+| `stg_sap__tvagt` | New model |  |  | Staging model for rejection reason descriptions (`tvagt`). |
+| `stg_sap__vbak` | New model |  |  | Staging model for sales document headers (`vbak`). |
+| `stg_sap__vbap` | New model |  |  | Staging model for sales document items (`vbap`). |
+| `stg_sap__vbuk` | New model |  |  | Staging model for sales document header status (`vbuk`). |
+| `stg_sap__vbup` | New model |  |  | Staging model for sales document item status (`vbup`). |
```

Comment:
I think we have to include the new tmp models too (see https://github.com/fivetran/dbt_shopify/releases/tag/v0.19.0), even though it is getting long in the tooth.